cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -315,10 +318,7 @@ def setUp(self):
         super(VideoUrlsCsvTestCase, self).setUp()
         VideoUploadConfig(
             profile_whitelist="profile1",
-            status_whitelist=(
-                status_display_string("file_complete") + "," +
-                status_display_string("transcode_active")
-            )
+            status_whitelist="file_complete,transcode_active"
```

Comment:
Yes, that's intentional. We shouldn't be putting display strings into the config.